### ISSUE TITLE
Not working with android facebook

### ISSUE THREAD

**Issue by NomadDevs** [leecrossley/cordova-plugin-social-message#36]
added text, image and url and when i click facebook nothing appears, it's just a new blank post.

**Comment by leecrossley** [comment]
Facebook message text for android is no longer possible, you can share a url or an image but not both. This is a limitation imposed by Facebook, not the plugin.

**Comment by NomadDevs** [comment]
So what is the one that breaks it? Can you not tell the script to only give FB the url and ignore the text because at the moment it tries to share nothing.

**Comment by loicfavory** [comment]
You have to modify the plugin to deal with Facebook.
If you're sharing an Intent with Facebook, you have to pass the URL in the Intent.EXTRA_TEXT

It's possible, I did it for one of my app : https://github.com/loicfavory/cordova-plugin-social-message

**Comment by leecrossley** [comment]
Can you clean and PR the modification @loicfavory?

**Comment by loicfavory** [comment]
@leecrossley I'm very busy this summer, but I'll try to find sometime this week end to do it. Did you check my code to see what I did ?

**Comment by leecrossley** [comment]
OK, I'll see if I can pull it across tomorrow - thanks

**Comment by loicfavory** [comment]
Not sure you can pull without cleaning it. I did a lot of specifics dev for my project.

**Comment by leecrossley** [comment]
Yes, of course. I will just take the relevant parts, pull was probably the wrong choice of word - thanks for helping out.

**Comment by loicfavory** [comment]
ok, I'm sorry, it's not well commented. Ask me if you need help but I'll not have a lot of time until saturday

**Comment by NomadDevs** [comment]
Hopefully this gets added before my release, thanks for the help both of you :)

**Comment by leecrossley** [comment]
so, are we saying that now we can only share a URL for Facebook Android? Your fork seems to suggest you ignore the other parameters in the case of Facebook.

**Comment by loicfavory** [comment]
I think you can only share a picture OR a link.
Facebook doesn't share text content.

For my project, I only need to share links, that why my fork ignore other parameters. 

Maybe we could allow to share picture or link, but I think it can't be both.

**Comment by NomadDevs** [comment]
Any update on this?

**Comment by issamdal** [comment]
Hello..I am using currently the plugin and sharing on facebook on android is important part of my app. will this issue be resolved soon? any workarounds? its a great plugin..I hope someone can help here

**Comment by loicfavory** [comment]
@leecrossley do you need help to solve this issue ?
I'm currently working on my project I could you some work for you if you need it.

**Comment by leecrossley** [comment]
Happy to take PRs

**Comment by loicfavory** [comment]
OK, seems there is a way to do it with picture & text using the Facebook SDK : http://stackoverflow.com/a/25065367
but I didn't test it, and will not have time now, but I'll work today on new branch to share only the URL as @NomadDevs asked
